src/definitions/collections/form.less

```diff
@@ -649,6 +649,11 @@
   height: 100%;
   z-index: @loaderDimmerZIndex;
 }
+
+.ui.loading.form.segments:before {
+  border-radius: @borderRadius;
```

Comment:
`@borderRadius` is not avaiable at that time. I think you are referring to that variable from `segments.variables`
So either add
```less
@borderRadius: @defaultBorderRadius;
```
to `form.variables`

or use

```less
border-radius: @defaultBorderRadius;
```

instead (because `@defaultBorderRadius` is available globally from `sites.variables`

Reply:
Oh, gotcha. I'll change that.